CIP-0068/README.md

```diff
@@ -1,77 +1,111 @@
 ---
 CIP: 68
 Title: Datum Metadata Standard
-Authors: Alessandro Konrad <alessandro.konrad@live.de>, Thomas Vellekoop <thomas.vellekoop@iohk.io>
-Comments-URI:
 Status: Proposed
-Type: Informational
+Category: Meta
+Authors: 
+  - Alessandro Konrad <alessandro.konrad@live.de>
+  - Thomas Vellekoop <thomas.vellekoop@iohk.io>
+Implementors:
```

Comment:
```suggestion
Implementors:
  - 
```

Reply:
other option = `Implementors: []` but please @Crypto2099 select one of these 😅